validator/project_validator_test.go

```diff
@@ -3240,6 +3240,12 @@ func TestValidateVersionControl(t *testing.T) {
 }
 
 func TestValidateContainers(t *testing.T) {
+	env := evergreen.GetEnvironment()
```

Comment:
Rather than setting the global state, I would change `validateContainers` to accept an environment and pass that through rather than set the global testing state.